.editorconfig

```diff
@@ -0,0 +1,165 @@
+# EditorConfig is awesome:

```

Comment:
we should put this on a public gist somewhere, so we don't have a 'source of truth' for various repos...

Reply:
But that would still mean we copy/ 🍝 the file around and don't know about changes to the source-of-truth file?

Reply:
yes, but we could link to that in each file for reference.

but more importantly we know where to source one from when we add a new one, as opposed to now. i'm guessing the processing now is, pick a random one in any repos we currently have cloned and use that? same thing occurs for .gitattributes and .gitignore...

all good, not blocking.. just wanted to flag it..

Reply:
EDIT: just reviewing the code now.. and i see you're doing what i want/prefer [here](response.Headers.Location.ShouldBe(new Uri($"http://localhost/test/{currentFakeVehicleCount + 1 }"));)

can we change this code to be like that?